tools/fonts/config/valid_codes.js

```diff
@@ -52,21 +52,51 @@ module.exports = {
   noteheadWhole: 'v1d',
   noteheadHalf: 'v81',
   noteheadBlack: 'vb',
+  noteheadXDoubleWhole: 'vxx',
   noteheadXWhole: 'v92',
   noteheadXHalf: 'v95',
   noteheadXBlack: 'v3e',
-  noteheadCircleX: 'v3b',
+  noteheadCircleXDoubleWhole: 'vxx',
+  noteheadCircleXWhole: 'vxx',
+  noteheadCircleXHalf: 'vxx',
+  noteheadCircleX: 'vxx',
```

Comment:
Can you explain the significance of 'vxx' vs null in this `valid_codes.js` file? I see that some of the fields are null, but others are 'vxx'. 

How come noteheadCircleX changes from v3b to vxx in this case?

Reply:
> Can you explain the significance of 'vxx' vs null in this `valid_codes.js` file? I see that some of the fields are null, but others are 'vxx'. 

In the generation process of the Gonville and Custom fonts they end up on different log messages:
- null -> No Glyph for...
- vxx -> No Outline for...
But there is no impact on the generated fonts. See fontgen-gonville-custom.js 

> How come noteheadCircleX changes from v3b to vxx in this case?

The size of this circlex head for Gonville does not much the size of the other durations' circlex heads in Bravura. I decided to disable it and fall back to Bravura for the complete set.

Reply:
The circle x, which is now smaller, and also `noteheadDiamondWhole`, which was pointing to a wrong Glyph, originates other visual difference in Gonville that I forgot to document (I only looked into Bravura).

NoteHead Drum_Chord_Heads Gonville
![NoteHead Drum_Chord_Heads Gonville](https://user-images.githubusercontent.com/22865285/172940980-11fc58e6-cd7e-41b2-a54b-d61d08d7abd6.png)
current
![NoteHead Drum_Chord_Heads Gonville_current](https://user-images.githubusercontent.com/22865285/172940983-b66d0a66-f1cf-41a6-b2a4-e591b79f91d3.png)
reference
![NoteHead Drum_Chord_Heads Gonville_reference](https://user-images.githubusercontent.com/22865285/172940984-cc1a1715-bbaa-4e25-8abd-291143b45959.png)
NoteHead Various_Heads Gonville
![NoteHead Various_Heads Gonville](https://user-images.githubusercontent.com/22865285/172941041-71ee8006-d2e5-4403-b616-15764445f642.png)
current
![NoteHead Various_Heads Gonville_current](https://user-images.githubusercontent.com/22865285/172941043-a8fb1cff-e5ad-4f98-8236-4f18ff8d19a6.png)
reference
![NoteHead Various_Heads Gonville_reference](https://user-images.githubusercontent.com/22865285/172941046-c4442834-ca0e-4010-99af-7db10bffe221.png)
Percussion Percussion_Basic2 Gonville
![Percussion Percussion_Basic2 Gonville](https://user-images.githubusercontent.com/22865285/172941092-3ee40bf0-8ce6-4a94-942c-b256054be8a5.png)
current
![Percussion Percussion_Basic2 Gonville_current](https://user-images.githubusercontent.com/22865285/172941094-81f627da-0310-4319-b1fc-8bb75a7a45e3.png)
reference
![Percussion Percussion_Basic2 Gonville_reference](https://user-images.githubusercontent.com/22865285/172941096-736485c9-b8b7-4d32-b399-73cc7f80d5b6.png)
Percussion Percussion_Notes Gonville
![Percussion Percussion_Notes Gonville](https://user-images.githubusercontent.com/22865285/172941100-2674a538-11b7-4fc8-b263-d7b5cbe32f8a.png)
current
![Percussion Percussion_Notes Gonville_current](https://user-images.githubusercontent.com/22865285/172941102-b68265fa-61d6-4c05-a8b3-9676cfa9cdc3.png)
reference
![Percussion Percussion_Notes Gonville_reference](https://user-images.githubusercontent.com/22865285/172941104-18e9ca72-364a-4f9f-97a3-dee6314114fb.png)
Percussion Percussion_Snare1 Gonville
![Percussion Percussion_Snare1 Gonville](https://user-images.githubusercontent.com/22865285/172941105-43512f80-d18f-4980-a085-363e9ca5d5f6.png)
current
![Percussion Percussion_Snare1 Gonville_current](https://user-images.githubusercontent.com/22865285/172941109-762763e9-4b97-4a34-818d-2ec45b33c7f2.png)
reference
![Percussion Percussion_Snare1 Gonville_reference](https://user-images.githubusercontent.com/22865285/172941114-2a4a47ef-fbbb-43aa-9fa4-7c95b9d23b24.png)